pkg/services/ngalert/schedule/schedule_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"time"
 
 	"github.com/grafana/grafana-plugin-sdk-go/data"
+
```

Comment:
I've seen a lot of variations of import grouping in the grafana codebase. Is there a preferred style - keepiong non-grafana, non-stdlib packages in their own block?

Reply:
It is not enforced by `gofmt` - but it is the preferred style.